libraries/core/src/daemon_messages.rs

```diff
@@ -85,6 +91,23 @@ impl DaemonRequest {
             | DaemonRequest::EventStreamDropped => true,
         }
     }
+
+    pub fn expects_tcp_json_reply(&self) -> bool {
+        #[allow(clippy::match_like_matches_macro)]
+        match self {
+            DaemonRequest::NodeConfig { .. } => true,
```

Comment:
Why is it not possible to send this as bincode too? Or is this just for making debugging easier?